src/main/java/eu/bbmri_eric/negotiator/governance/resource/ResourceController.java

```diff
@@ -43,4 +52,20 @@ public PagedModel<EntityModel<ResourceResponseModel>> list(@Nullable ResourceFil
     return resourceModelAssembler.toPagedModel(
         (Page<ResourceResponseModel>) resourceService.findAll(filters), filters);
   }
+
+  @PostMapping
+  @Operation(summary = "Add a list of resources")
+  @ResponseStatus(HttpStatus.CREATED)
+  public Collection<EntityModel<ResourceResponseModel>> addResources(
```

Comment:
please add content types

Reply:
Fixed in 1353b5e7908df5d76ea043a1570702aeceaffc67